include/ppx/command_line_parser.h

```diff
@@ -64,11 +66,17 @@ class CliOptions
             return defaultValue;
         }
         auto valueStr = it->second.back();
-        return GetParsedOrDefault<T>(valueStr, defaultValue);
+        T    parsedValue;
+        auto error = ppx::string_util::Parse(valueStr, parsedValue);
+        if (error != std::nullopt) {
+            PPX_LOG_ERROR(error->errorMsg);
+            return defaultValue;
+        }
+        return parsedValue;
     }
 
     // Same as above, but intended for list flags that are specified on the command line
-    // with multiple instances of the same flag
+    // with multiple instances of the same flag, or with comma-separated values
```

Comment:
I vaguely remember @Keenuts suggesting we stick to one flag one value for simplicity, and since "," can be a valid name on Linux, and a similar change was removed in a previous PR. Is there any reason to add it now? 
Also based on brief testing: 
./program --assets-path=A,B --assets-path=C    => (A,B,C)
./program --assets-path=A,B --assets-path=C,D => (A,B)        and C,D are ignored
So only the first one can be comma separated?

Reply:
I see. I think it is fine to allow comma separated values, but I would at least want to consistent and predictable. Seems like right now if the first value is comma separated, then subsequent comma separated values are ignored? 
Also fwiw, I don't think it would be very meaningful for assets paths to be modified at runtime at the moment, since most are loaded at startup, but I guess there would be other uses of list of values.